helm-chart/metabase/templates/deployment.yaml

```diff
@@ -0,0 +1,872 @@
+          - name: MB_EMAIL_SMTP_HOST
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-secrets
+                key: MB_EMAIL_SMTP_HOST
+          - name: MB_EMAIL_SMTP_USERNAME
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-secrets
+                key: MB_EMAIL_SMTP_USERNAME
+          - name: MB_EMAIL_SMTP_PASSWORD
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-secrets
+                key: MB_EMAIL_SMTP_PASSWORD
+          - name: MB_EMAIL_SMTP_PORT
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-secrets
+                key: MB_EMAIL_SMTP_PORT
+          - name: MB_EMAIL_SMTP_SECURITY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-secrets
+                key: MB_EMAIL_SMTP_SECURITY
+          {{- end }}
+          {{- if .Values.config.security.passComplexity }}
+          - name: MB_PASSWORD_COMPLEXITY
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MB_PASSWORD_COMPLEXITY
+          {{- end }}
+          {{- if .Values.config.security.passLength }}
+          - name: MB_PASSWORD_LENGTH
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MB_PASSWORD_LENGTH
+          {{- end }}
+          {{- if .Values.config.dw.maxConnectionPoolSize }}
+          - name: MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MB_APPLICATION_DB_MAX_CONNECTION_POOL_SIZE
+          {{- end }}
+          {{- if .Values.config.email.fromAddress }}
+          - name: MB_EMAIL_FROM_ADDRESS
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MB_EMAIL_FROM_ADDRESS
+          {{- end }}
+          {{- if .Values.config.security.sessionAge }}
+          - name: MAX_SESSION_AGE
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MAX_SESSION_AGE
+          {{- end }}
+          {{- if .Values.config.email.admin }}
+          - name: MB_ADMIN_EMAIL
+            valueFrom:
+              configMapKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
+                key: MB_ADMIN_EMAIL
+          {{- end }}
+          {{- if .Values.secrets.config.api_key }}
+          - name: MB_API_KEY
+            valueFrom:
+              secretKeyRef:
+                name: {{ include "metabase.fullname" . }}-configs
```

Comment:
this should be the secrets not the configmap

Reply:
good catch